cime_config/config_compsets.xml

```diff
@@ -32,17 +32,36 @@
       - lname
       - alias
       - support  (optional description of the support level for this compset)
+
     Each compset node can also have the following attributes
       - grid  (optional regular expression match for grid to work with the compset)
   </help>
 
+
+  <!-- NOTE: the NorESM physics does not work with cam60 - so must use camdev if noresm physics is used  -->
+
   <!-- ******************************** -->
   <!-- CAM-Nor compsets                 -->
   <!-- ******************************** -->
 
-  <compset>
-    <alias>NF2000dev</alias>
-    <lname>2000_CAM%DEV%LT%NORESM_CLM50%SP_CICE%PRES_DOCN%DOM_MOSART_SGLC_SWAV</lname>
+  <!-- CAM60 - the following currently do not work -->
+  <!-- <compset> -->
+  <!--   <alias>NF2000climoC6</alias> -->
+  <!--   <lname>2000_CAM60%NORESM_CLM50%SP_CICE%PRES_DOCN%DOM_MOSART_SGLC_SWAV</lname> -->
+  <!-- </compset> -->
+  <!-- <compset> -->
+  <!--   <alias>NQPC6</alias> -->
+  <!--   <lname>2000_CAM60%NORESM_SLND_SICE_DOCN%AQP3_SROF_SGLC_SWAV</lname> -->
+  <!-- </compset> -->
+  <!-- <compset> -->
+  <!--   <alias>NQPC2000climoC6</alias> -->
+  <!--   <lname>2000_CAM60%NORESM%CCTS1_SLND_SICE_DOCN%AQP3_SROF_SGLC_SWAV</lname> -->
+  <!-- </compset> -->
```

Comment:
I'm not sure these are ever going to work, why not just remove them for now?

Reply:
I have removed them.